Figure readme updates (#24)
greenelab/annorxiver: README.md

# Annorxiver

## Citation

> - [Manuscript in progress](greenelab.github.io/annorxiver_manuscript)

Reviewer: ```suggestion
- [Manuscript in progress](https:/greenelab.github.io/annorxiver_manuscript)
```

Reviewer: Link didn't work, but i think this should fix it. Tested it already

- doi and full citation coming soon!!

## About

> This repository contains code for the annorxvier project.

Reviewer 2: ```suggestion
This repository contains code for the annorxiver project.
```

> This project focuses on analyzing the linguistic style and content of bioRxiv preprints and understanding how these features change when preprints undergo the publication process.

Reviewer: ```suggestion
This project analyzes the linguistic style and content of bioRxiv preprints and aims to understand how these features change when preprints undergo the publication process.
```

We are currently working on a manuscript for the work in this repository.

> Feel free to check it out! (link above).

Reviewer: Not sure if you will keep this sentence in the future, but if you do, maybe just add the link here `(link above)` >> `(link [here](https:/greenelab.github.io/annorxiver_manuscript)`

> We also created a [web app](greenelab.github.io/preprint-similarity-search) allows anyone to input a bioRxiv or medRxiv preprint and receive a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.

Reviewer: ```suggestion
We also created a [web app](greenelab.github.io/preprint-similarity-search) that takes a bioRxiv or medRxiv preprint doi as input and returns a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.
```

## Directory Structure
| Folder/file | Description |
| --- | --- | 

> | [annorxiver_modules](annorxiver_modules) | This folder contains universal functions that other notebooks in this repository will use |

Reviewer: ```suggestion
| [annorxiver_modules](annorxiver_modules) | This folder contains supporting functions that other notebooks in this repository will use |
```

| [biorxiv](biorxiv) | This folder contains all experiments that are related to biorxiv preprints. | 

> | [credentials](credentials) | Some of the code in this repository needs credentials to run. Any users that need to run those notebooks use check this folder first. |

Reviewer: ```suggestion
| [credentials](credentials) | Some of the code in this repository need credentials to run. Any user that needs to run those notebooks should check this folder first. |
```

Reviewer: Do all notebooks need a user to fill out credentials? I assume this just pertains to the download of the data. If so, maybe you could add a separate section for how to run the experiments.

| [figure_generation](figure_generation) | This folder contains code to generate figures for the manuscript in progress. |
| [nytac/corpora_stats](nytac/corpora_stats) | This folder contains results when parsing the New York Times annotated corpus (NYTAC) from the Linguistic Data Consortium (LDC). |
| [pmc](pmc) |  This folder contains all experiments that are related to articles in Pubmed Central Open Access corpus (PMC). | 

> | environment.yml | This file contains the necessary packages this repository uses.  |

Reviewer: Do you want to add links to these files too?

| setup.py | This file sets up the annorxiver modules to be used as a regular python package. |

## Set up Environment

> Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://www.continuum.io/downloads)

Reviewer: The continuum link seems to be broken

> Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://www.continuum.io/downloads)

Reviewer 2: ```suggestion
Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://docs.conda.io/en/latest/miniconda.html)
```

> Once everything has been installed, type following command in the terminal: 

Reviewer 2: May want to say "Navigate to the annorxiver directory and type the following command in the terminal"
